### DISCUSSION TITLE
GraphQL setup endpoint

### DISCUSSION THREAD

**Discussion by EugeneGohh** [community/community#24739] [Apps, API and Webhooks]
<p>Hi guys, I am having a problem with GraphQL. Can anyone tell me where to set this up? It is <strong>GraphQL with client course on <a href="http://educative.io" rel="noopener nofollow ugc">educative.io</a></strong>.</p>
<p>And this is the course article:</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/64187129/181086513-98436d60-cbcf-4d4f-8174-a436c5dcf28c.png" class="site-icon" width="32" height="32">

      <a href="https://www.robinwieruch.de/getting-started-github-graphql-api/" target="_blank" rel="noopener nofollow ugc">rwieruch</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/460;"><img src="https://user-images.githubusercontent.com/64187129/181086514-4737a0d3-b3e8-4a45-9c2a-d625cf84628e.jpeg" class="thumbnail" width="690" height="460" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85afd39524b0e79d09f107f26bf6b3fc8edc02aa_2_690x460.jpeg, https://aws1.discourse-cdn.com/github/original/3X/8/5/85afd39524b0e79d09f107f26bf6b3fc8edc02aa.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/5/85afd39524b0e79d09f107f26bf6b3fc8edc02aa.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85afd39524b0e79d09f107f26bf6b3fc8edc02aa_2_10x10.png"></div>

<h3><a href="https://www.robinwieruch.de/getting-started-github-graphql-api/" target="_blank" rel="noopener nofollow ugc">Getting Started with GitHub's GraphQL API - RWieruch</a></h3>

  <p>Everything you need to get started with GitHub's GraphQL API for learning GraphQL in JavaScript. This walkthrough demonstrates you how to set up your data on GitHub, how to create a personal access token, and how to consume their API without any...</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>It said to set this endpoint in GraphiQL. I can’t find it.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/d/7d2140637af26c4a7ef5a114d74e610ba35c9c4d.png" data-download-href="/uploads/short-url/hQWVl8MURetvHT0SsJz1zXTadFH.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/64187129/181086516-f9ce02b9-f34a-42f9-aba0-ef3704ce65d8.png" alt="image" data-base62-sha1="hQWVl8MURetvHT0SsJz1zXTadFH" width="690" height="62" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d2140637af26c4a7ef5a114d74e610ba35c9c4d_2_690x62.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d2140637af26c4a7ef5a114d74e610ba35c9c4d_2_1035x93.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d2140637af26c4a7ef5a114d74e610ba35c9c4d_2_1380x124.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d2140637af26c4a7ef5a114d74e610ba35c9c4d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1502×136 52.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Have you installed the <a href="https://github.com/skevy/graphiql-app">Graphiql app</a>?</p>
<p>You should be able to save your endpoint and HTTP headers exactly as shown.</p>

**Comment by EugeneGohh** [comment]
<p>Thank you so much, my problem is solved. I have installed GraphiQL standalone application via here <a href="https://www.electronjs.org/apps/graphiql" rel="noopener nofollow ugc">Electron</a>. The documentation didn’t specifically mention Windows installation so does the course didn’t provide the correct link.</p>
